docs.openc3.com/docs/getting-started/installation.md

```diff
@@ -11,7 +14,7 @@ The following sections describe how to get OpenC3 COSMOS installed on various op
 
 ### PREREQUISITES
 
-Install [Docker](https://docs.docker.com/get-docker/) and install [Docker Compose](https://docs.docker.com/compose/install/).
+If you're on linux (recommended for production), we recommend installing Docker using your native package manager. Note: Red Hat users should read the [Podman](podman) documentation. If you're on Windows or Mac, install [Docker Desktop](https://docs.docker.com/get-docker/). All platforms also need to install [Docker Compose](https://docs.docker.com/compose/install/).
```

Comment:
Added a note to clarify using native Docker on linux and only Docker Desktop on Mac and Windows